.github/workflows/ssi-authority-schema-registry-chart-test.yml

```diff
@@ -0,0 +1,119 @@
+        # k8s version from 3.1 release as default
+        default: 'kindest/node:v1.27.3'
+        required: false
+        type: string
+      upgrade_from:
+        description: 'ssi authority & schema registry chart version to upgrade from'
+        # tbd
+        default: '0.1.0-rc.1'
+        required: false
+        type: string
+
+jobs:
+
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Kubernetes KinD Cluster
+        uses: container-tools/kind-action@v1
+        with:
+          # upgrade version, default (v0.20.0) uses node image v1.27.3 and doesn't work with more recent node image versions
+          version: v0.20.0
+          # default value for event_name != workflow_dispatch
+          node_image: ${{ github.event.inputs.node_image || 'kindest/node:v1.27.3' }}
+
+      - name: Build migration image
+        id: build-migration-image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: docker/Dockerfile-ssi-authority-schema-registry-migrations
+          push: true
+          tags: kind-registry:5000/ssi-authority-schema-registry-migrations:testing
+
+      - name: Build service image
+        id: build-service-image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: docker/Dockerfile-ssi-authority-schema-registry-service
+          push: true
+          tags: kind-registry:5000/ssi-authority-schema-registry-service:testing
+  
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.9.3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.3.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
+          if [[ -n "$changed" ]]; then
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
+
+      - name: Run service chart-testing (install)
+        run: ct install --charts charts/ssi-authority-schema-registry --config charts/chart-testing-config.yaml --helm-extra-set-args "--set=ssiauthorityschemaregistry.image=kind-registry:5000/ssi-authority-schema-registry-service:testing --set=ssiauthorityschemaregistrymigrations.image=kind-registry:5000/ssi-authority-schema-registry-migrations:testing"
+        if: github.event_name != 'pull_request' || steps.list-changed.outputs.changed == 'true'
+
+        # currently the update is commented out, at the moment we're working for the initial version, after that the update will be enabled
```

Comment:
could you please add the updated version from here?m https://github.com/eclipse-tractusx/policy-hub/blob/release/v0.1.0-rc.3/.github/workflows/policy-hub-chart-test.yml#L111

Reply:
adjusted